mythril/interfaces/cli.py

```diff
@@ -275,6 +278,24 @@ def main():
                 "The --query-signature function requires the python package ethereum-input-decoder",
             )
 
+    if args.enable_iprof:
+        ignore_iprof = True
+
+        if args.v < 4:
+            exit_with_error(
+                args.outform,
+                "--enable-iprof must be used with -v LOG_LEVEL where LOG_LEVEL >= 4",
+            )
+        elif not (args.graph or args.fire_lasers or args.statespace_json):
+            exit_with_error(
+                args.outform,
+                "--enable-iprof must be used with one of -g, --graph, -x, --fire-lasers, -j and --statespace-json",
+            )
+        else:
+            ignore_iprof = False
+
+        args.enable_iprof = not ignore_iprof
```

Comment:
if you reach this position then ignore_iprof is always false, right? 
I think with the exit_with_error, ln 294,295,282 and 297 are not necessary anymore.